backend/src/tests/mock_builders/playback_info_builder.py

```diff
@@ -0,0 +1,28 @@
+from src.dataclasses.playback_info import PlaybackInfo
+
+
+def playback_info_builder(
```

Comment:
These sort of tests do scale well, but they are annoying to write initially - well done for seeing it through! 😄